core/src/main/java/com/linecorp/armeria/common/logging/TextLogFormatter.java

```diff
@@ -182,7 +182,7 @@ public String formatRequest(RequestOnlyLog log) {
             if (sanitizedTrailers != null) {
                 buf.append(", trailers=").append(sanitizedTrailers);
             }
-            buf.append('}');
+            buf.append(", currentAttempt=").append(log.currentAttempt()).append('}');
```

Comment:
Should we not add this when `currentAttempt == 0`? I'm fine with the current implementation, though.

Reply:
I also prefer not to include it to make the log more concise. 😉